content/en/monitors/guide/integrate-monitors-with-statuspage.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Integrating Monitors With Statuspage
```

Comment:
```suggestion
title: Integrating Monitors with Statuspage
```

Reply:
No clear guidance on this, but it looks like we don't capitalize prepositions or articles in our titles.